### DISCUSSION TITLE
Using C++ with Codespaces?

### DISCUSSION THREAD

**Discussion by 2percentsilk** [community/community#3413] [Codespaces]
Hey C++ GitHub Codespace developers! 

We’d love to learn more about your experience working with C++ projects in Codespaces. Please use this thread to tell us the good, the bad, or any suggestions you have specific to C++ development.

**Comment by PranamBhat** [comment]
Hi @2percentsilk 

one quick question.

Is this **Codespaces Feedback** is going to be language specific category in coming days?

**Reply by 2percentsilk** [reply]
Nope - this is just one discussion I opened to get feedback from users of a specific language. You'll likely see discussions pop up for various stacks for Codespaces in the future 😄

**Reply by PranamBhat** [reply]
**Perfect! I was scared!!!**
